src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java

```diff
@@ -159,17 +158,16 @@ public void should_record_signature_failure_when_zip_contains_invalid_signature(
 
         // then
         envelopeWasNotCreated();
-        eventWasCreated("43_24-06-2018-00-00-00.test.zip", Event.DOC_SIGNATURE_FAILURE);
+        eventWasCreated(Event.DOC_SIGNATURE_FAILURE);
     }
 
-    private void eventWasCreated(String zipFileName, Event event) {
+    private void eventWasCreated(Event event) {
         List<ProcessEvent> processEvents = processEventRepository.findAll();
         assertThat(processEvents).hasSize(1);
 
         ProcessEvent processEvent = processEvents.get(0);
 
         assertThat(processEvent.getContainer()).isEqualTo(testContainer.getName());
-        assertThat(processEvent.getZipFileName()).isEqualTo(zipFileName);
```

Comment:
'zip file name' field on event is set based on the the metadata json field, not the zip file name.